src/BinSkim.Rules/PERules/BA2024.EnableSpectreMitigations.cs

```diff
@@ -308,10 +322,28 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
             string line;
             var sb = new StringBuilder();
 
+            bool mitigationMissingCommandLineUnknownOnly = false;
+
+            if (mitigationUnknownNoCommandLineRaw.Count > 0)
+            {
+                mitigationMissingCommandLineUnknownOnly = true;
+
+                // The following modules were compiled with a toolset that supports
+                // /Qspectre but no compiler command line is present and therefore
+                // it cannot be determined if /Qspectre was specified: {0}
+                line = string.Format(
+                RuleResources.BA2024_Warning_SpectreMitigationUnknownNoCommandLine,
+                        mitigationUnknownNoCommandLineRaw.CreateOutputCoalescedByLibrary());
+                sb.AppendLine(line);
+            }
+
             if (mitigationExplicitlyDisabledModules.Count > 0)
             {
-                // The following modules were compiled with Spectre
-                // mitigations explicitly disabled: {0}
+                mitigationMissingCommandLineUnknownOnly = false;
+
+                // The following modules were compiled with a toolset that supports
+                // /Qspectre but no compiler command line is present and therefore
+                // it cannot be determined if /Qspectre was specified: {0}
                 line = string.Format(
                         RuleResources.BA2024_Warning_SpectreMitigationExplicitlyDisabled,
```

Comment:
>BA2024_Warning_SpectreMitigationExplicitlyDisabled [](http://example.com/codeflow?start=38&length=50)

the comment here "but no compiler command line is present and therefore it cannot be determined if /Qspectre was specified"
does not match the code:
`BA2024_Warning_SpectreMitigationExplicitlyDisabled`

Reply:
Nice catch.
Fixed.